x/auth/client/errors.go

```diff
@@ -1,4 +1,4 @@
-package utils
+package client
 
 import "errors"
```

Comment:
these errors should be defined in `auth/types/errors.go` instead

Reply:
Done